pyproject.toml

```diff
@@ -28,17 +28,17 @@ keywords = [
 exclude = ["tests/"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-Flask = "^1.1.2"
-Flask-SQLAlchemy = "^2.4.1"
-python-decouple = "^3.3"
-SQLAlchemy = "^1.3.16"
+python = ">=3.6.2,<4.0"
+Flask = ">=1.1.2"
```

Comment:
We should lock it to versions up to 2.x.x, so we don't accidentally install it in Flask 3 in the future (just as you did with Python).